Public/Src/Cache/ContentStore/Distributed/Stores/DistributedContentCopier.cs

```diff
@@ -171,7 +171,16 @@ protected override Task<BoolResult> ShutdownCoreAsync(OperationContext context)
                     }
 
                     TimeSpan waitDelay = _retryIntervals[attemptCount];
-                    Tracer.Warning(operationContext, $"{AttemptTracePrefix(attemptCount)} All replicas {hashInfo.Locations.Count} failed. Retrying for hash {hashInfo.ContentHash}...");
+                    bool extendedForBadLocations = false;
+
+                    if (badContentLocations.Count == hashInfo.Locations.Count)
+                    {
+                        // Double delay because all locations are bad. This allows the machines more time to recover, especially if they are overwhelmed by requests.
+                        waitDelay += TimeSpan.FromTicks((long)(waitDelay.Ticks * ThreadSafeRandom.Generator.NextDouble()));

```

Comment:
In the first few iterations, this will actually wait for -strictly- less than we would have originally, which would only make the problem worse. I'm not sure how this interacts with the other behaviors of this object, but I'm thinking `attemptCount = 2;` should work? so as to enter the exponential decay behavior directly.

Reply:
Wouldn't this change increase `waitDelay` by to up-to double the current delay? Note the `+=` operator.

I'm worried that increasing `attemptCount` would skip useful retries.

Reply:
After discussing offline, I agree that reducing the delay in this case is acceptable. The new delay's randomized distribution is `[0.5 * delay, delay)`.